calibrated: 0
signEquilibrium: -1
signFlexor: -1
signExtensor: -1
signEncoder: -1
signJoint: 1
name: "wrist"
nameFlexor: "wrist_flexor"
nameExtensor: "wrist_extensor"
nameEncoder: "wrist_encoder"
servoRange: 5.4978
servoOffset: 0
minAngle: -0.9
maxAngle: 0.9
angleOffset: 0.0
gains:
    P: 0.025
    D: 0.015